docs/console.rst

```diff
@@ -0,0 +1,181 @@
+Console
+=======
+
+.. autoclass:: ev3dev2.console.Console
+    :members:
+
+Examples:
+
+.. code-block:: py
+
+    #!/usr/bin/env micropython
+    from ev3dev2.console import Console
+
+    # create a Console instance, which uses the default font, clears the screen,
+    # and turns off the cursor
+    console = Console()
+
+    # display 'Hello World!' at row 5, column 1 in inverse, but clear the console first
+    console.text_at('Hello World!', row=5, column=1, clear_screen=True, inverse=True)
+
+.. code-block:: py
+
+    #!/usr/bin/env micropython
+    from time import sleep
+    from ev3dev2._platform.ev3 import INPUT_1, INPUT_2, INPUT_3
+    from ev3dev2.console import Console
+    from ev3dev2.sensor.lego import GyroSensor, ColorSensor
+
+    console = Console()
+    gyro = GyroSensor(INPUT_1)
+    gyro.mode = GyroSensor.MODE_GYRO_ANG
+    color_sensor_left = ColorSensor(INPUT_2)
+    color_sensor_right = ColorSensor(INPUT_3)
+
+    # show the gyro angle and reflected light intensity for both of our color sensors
+    while True:
+        angle = gyro.angle
+        left = color_sensor_left.reflected_light_intensity
+        right = color_sensor_right.reflected_light_intensity
+
+        # show angle in inverse color when pointing at 0
+        console.text_at("G: %03d" % (angle), column=5, row=1, clear_screen=True, text_color=(angle == 0))
+
+        # show light intensity values in inverse when 'dark'
+        console.text_at("L: %02d" % (left), column=0, row=3, clear_screen=False, inverse=(left < 10))
+        console.text_at("R: %02d" % (right), clear_screen=False, x=10, y=3, inverse=(right < 10))
+
+        sleep(0.5)
+
+Console fonts
+-------------
+
+The :py:class:`ev3dev2.console.Console` class displays text on the LCD console
+using ANSI codes in various system console fonts. The system console fonts are
+located in `/usr/share/consolefonts`.
+
+Font filenames consist of the codeset, font face and font size. The codeset
+specifies the characters supported. The font face determines the look of the
+font. Each font face is available in multiple sizes.
+
+For Codeset information, see
+`<https://www.systutorials.com/docs/linux/man/5-console-setup/#lbAP>`.
```

Comment:
Since you have this link that covers the next ~40 lines of info I would chop the ~40 lines of info...it is a lot of text that most folks won't read.

Reply:
Ok. Agreed.